curriculum/challenges/english/25-front-end-development/quiz-loops-and-sequences/67f41268a129e63f8e071736.md

```diff
@@ -17,439 +17,522 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
 
 #### --text--
 
-Placeholder question
+Which of the following is the correct syntax for a list?
 
 #### --distractors--
 
-Placeholder distractor 1
+```py
+cities = //'Los Angeles', 'London', 'Tokyo'\\
+```
 
 ---
 
-Placeholder distractor 2
+```py
+cities = {'Los Angeles', 'London', 'Tokyo'}
+```
 
 ---
 
-Placeholder distractor 3
+```py
+cities = <'Los Angeles', 'London', 'Tokyo'>
+```
 
 #### --answer--
 
-Placeholder answer
+```py
+cities = ['Los Angeles', 'London', 'Tokyo']
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will be printed to the console?
+
+```py
+cities = ['Los Angeles', 'London', 'Tokyo']
+print(cities[-1]) # ?
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+`'None'`
```

Comment:
Not sure if these should include the single quotes in these four answers or not. I would vote no, but they're fine if we want them.

Reply:
yeah, I think the strings are fine here